doc/lua_api.txt

```diff
@@ -3115,7 +3115,15 @@ For the following functions, `v`, `v1`, `v2` are vectors,
 * `vector.cross(v1, v2)`:
     * Returns the cross product of `v1` and `v2`.
 * `vector.offset(v, x, y, z)`:
-    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.
+    * Returns the sum of the vectors `v` and `{x = x, y = y, z = z}`.#
+* `vector.to_string(v)`:
+    * Returns a string `"("..v.x..", "..v.y..", "..v.z..")"`.
+* `vector.from_string(s[, init])`:
+    * Returns a new vector by reading it from the given string `s`.
+    * Returns `nil` on failure.
+    * `s`: Is of the form `"(x, y, z)"`. Additional spaces, leaving away commas
+           and adding an additional comma to the end is allowed.
+    * `init`: If given starts looking for the vector at this string index.
```

Comment:
Since `init` can be specified, it might also be good to return the position after the vector in the string so multiple vectors in the same string can be read.  So, `vector.from_string("a (1,2,3) s (4,5,6)", 3)` would return `{x=1, y=2, z=3}, 10`.

Reply:
Nice idea! Will do.